src/libraries/Common/src/System/Collections/Generic/BitHelper.cs

```diff
@@ -19,19 +19,21 @@ internal BitHelper(Span<int> span, bool clear)
 
         internal void MarkBit(int bitPosition)
         {
```

Comment:
Can we add a `Debug.Assert(bitPosition >= 0)`? Same for the other two methods here.

Reply:
It might even be better to just keep this one "as is" since its really doing the same thing, just with more casts now (that is comparing `uint` to `uint`)